har2locust/default_plugins/rest.py

```diff
@@ -13,7 +14,7 @@ def process(entries: list[dict]):
                         "postData" not in r  # not a post, ok for .rest
                         or r["postData"]["mimeType"] == "application/json"  # json payload, also ok
                     ):
-                        logging.debug(f"{r['url']} is a rest request!")
+                        # logging.debug(f"{r['url']} is a rest request!")
```

Comment:
Det här kanske borde bort helt?

Reply:
maybe. I wish there was a built in trace-level logging.